### DISCUSSION TITLE
EF66581B-95AA-3049-70CD-BACB62E89680

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#39] [General]
# EF66581B-95AA-3049-70CD-BACB62E89680

This is a slightly modified and resurrected basic package of Jestr's Star Wars add-on. It contains Tatooine, Endor and Death Star. This package is required for the most other Star wars add-ons by Jestr. The following modifications were mady by Ulrich Dickmann (Adirondack) in December 2012 (corrected folder structure, fixed ssc file).
Updated on 6/20/2022 to change the star of the Addon to a fictional star named Jestr's Star (named after the original addon creator.)
Original release date of unedited addon is 2004.

https://celestia.mobi/addon?item=EF66581B-95AA-3049-70CD-BACB62E89680&lang=en

**Comment by Dirgham2015** [comment]
Ooohhh
